dev-config/PSQL_CONNECTION_STRING

```diff
@@ -1 +1 @@
-postgresql+psycopg2://progcom:progcom@localhost:5432/progcom
+postgresql+psycopg2://test:test@localhost:5432/test
```

Comment:
I'm not sure this change should necessarily be in this PR. Is `progcom` the production user?